src/qibo/backends/__init__.py

```diff
@@ -36,11 +36,7 @@ def construct_backend(backend, **kwargs):
 
         return QibolabBackend(**kwargs)
     elif backend == "clifford":
-        platform = kwargs.get("platform")
-        if platform in ("cupy", "numba", "cuquantum"):
-            platform = construct_backend("qibojit", platform=platform)
-        elif platform == "numpy":
-            platform = construct_backend(platform)
```

Comment:
@BrunoLiegiBastonLiegi I'm not sure why you removed this...

Reply:
I assumed it was an error due to merging an old version of the `master` branch, and I reintroduced it.

If I was wrong, just let me know.

Reply:
Mmmh yes I think there was some mess due to the master merge, but I don't think is needed anymore as now we only pass the name of the engine as a string. At least in master it's like that.

Reply:
Ok, then I converted `platform` in `kwargs["platform"]`. Maybe it was like that, and somehow changed during the merge.

(I only had a look to the diff, so I did not realize before, sorry)